Source/ChocolateyGui/Properties/Resources.sv-SE.resx

```diff
@@ -0,0 +1,357 @@
+    <value>Nästa</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGotoFirstPage" xml:space="preserve">
+    <value>Gå till första sidan</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGotoLastPage" xml:space="preserve">
+    <value>Sista</value>
+  </data>
+  <data name="RemoteSourceView_ButtonRefreshPkgs" xml:space="preserve">
+    <value>Kontrollera paket</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxAllVersions" xml:space="preserve">
+    <value>Alla versioner</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxIncludePrerelease" xml:space="preserve">
+    <value>Inklusive förhandsversioner</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxMatchExactly" xml:space="preserve">
+    <value>Matcha ord exakt</value>
+  </data>
+  <data name="RemoteSourceView_Downloads" xml:space="preserve">
+    <value>nedladdningar</value>
+  </data>
+  <data name="RemoteSourceView_SearchBoxText" xml:space="preserve">
+    <value>Sök:</value>
+  </data>
+  <data name="Resources_ThisPC" xml:space="preserve">
+    <value>Den här datorn</value>
+  </data>
+  <data name="SettingsView_About" xml:space="preserve">
+    <value>Om</value>
+  </data>
+  <data name="SettingsView_ButtonCancel" xml:space="preserve">
+    <value>Avbryt</value>
+  </data>
+  <data name="SettingsView_ButtonNew" xml:space="preserve">
+    <value>Ny</value>
+  </data>
+  <data name="SettingsView_ButtonRemove" xml:space="preserve">
+    <value>Ta bort</value>
+  </data>
+  <data name="SettingsView_ButtonSave" xml:space="preserve">
+    <value>Spara</value>
+  </data>
+  <data name="SettingsView_ChocoGui" xml:space="preserve">
+    <value>Chocolatey Gui</value>
+  </data>
+  <data name="SettingsView_Credits" xml:space="preserve">
+    <value>Tack till</value>
+  </data>
+  <data name="SettingsView_Features" xml:space="preserve">
+    <value>Funktioner</value>
+  </data>
+  <data name="SettingsView_Options" xml:space="preserve">
+    <value>Alternativ</value>
+  </data>
+  <data name="SettingsView_PropertyDescription" xml:space="preserve">
+    <value>Beskrivning</value>
+  </data>
+  <data name="SettingsView_PropertyName" xml:space="preserve">
+    <value>Namn</value>
+  </data>
+  <data name="SettingsView_PropertyValue" xml:space="preserve">
+    <value>Värde</value>
+  </data>
+  <data name="SettingsView_ReleaseNotes" xml:space="preserve">
+    <value>Ändringslogg</value>
+  </data>
+  <data name="SettingsView_Settings" xml:space="preserve">
+    <value>Inställningar</value>
+  </data>
+  <data name="SettingsView_Sources" xml:space="preserve">
+    <value>Källor</value>
+  </data>
+  <data name="SettingsView_SourcesCertificate" xml:space="preserve">
+    <value>Certifikat</value>
+  </data>
+  <data name="SettingsView_SourcesCertificatePass" xml:space="preserve">
+    <value>Certifikatlösenord</value>
+  </data>
+  <data name="SettingsView_SourcesId" xml:space="preserve">
+    <value>Id</value>

```

Comment:
I think we should keep the casing as in the English source text where applyable.